docs/source/quickstart.md

```diff
@@ -327,15 +291,10 @@ man = Maneuver.impulse(dv)
 man = Maneuver((0 * u.s, dv))  # Equivalent
 ```
 
-```{eval-rst}
-There are other useful methods you can use to compute common in-plane
-maneuvers, notably :py:meth:`~poliastro.maneuver.Maneuver.hohmann` and
-:py:meth:`~poliastro.maneuver.Maneuver.bielliptic` for `Hohmann`_ and
-`bielliptic`_ transfers respectively. Both return the corresponding
-``Maneuver`` object, which in turn you can use to calculate the total cost
-in terms of velocity change :math:`\sum |\Delta v_i|` and the transfer
-time:
-```
+
+There are other useful methods you can use to compute common in-plane maneuvers, notably {py:meth} `~poliastro.maneuver.Maneuver.hohmann` and {py:meth}`~poliastro.maneuver.Maneuver.bielliptic` for `Hohmann`_ and `bielliptic`_ transfers respectively. Both return the corresponding ``Maneuver`` object, which in turn you can use to calculate the total cost
```

Comment:
I wonder if this

```
`Hohmann`_
```

syntax works? 🤔 Or is it reStructuredText? I think there is another way of doing this in MyST